go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/grafana/grafana-plugin-sdk-go v0.129.0
 	github.com/grafana/loki v1.6.2-0.20211015002020-7832783b1caa
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
-	github.com/hashicorp/go-hclog v0.16.1
+	github.com/hashicorp/go-hclog v1.0.0
```

Comment:
@marefr are we able to safely upgrade this now?  I remember issues a while back, but not sure if there still exists any good reason

Reply:
Yeah not sure, could have some unexpected side-effects with logs coming from backend plugins ([code](https://github.com/grafana/grafana/blob/main/pkg/plugins/backendplugin/grpcplugin/log_wrapper.go)). Please revert if you can, otherwise let us know we can help out verify if there's an actual problem.

Reply:
had to do a replace because ridiculousness of modules, but it's fixed